source/MissionAction.cpp

```diff
@@ -430,6 +430,14 @@ bool MissionAction::CanBeDone(const PlayerInfo &player, const shared_ptr<Ship> &
 				{
 					available += ship->Cargo().Get(it.first);
 					available += ship->OutfitCount(it.first);
+					
+					// Check for this outfit's variants
+					if (GameData::OutfitVariants().find(it.first) != GameData::OutfitVariants().end())
```

Comment:
Read up on the [style guide](https://endless-sky.github.io/styleguide/styleguide.xml); use `if(`, not `if (`, and comments should be formatted like statements (i.e. add periods to the end of your comments).